rooms.js

```diff
@@ -932,17 +932,25 @@ class BattleRoom extends Room {
 		this.resetUser = '';
 	}
 	requestKickInactive(user, force) {
-		if (this.resetTimer) {
-			if (user) this.sendUser(user, '|inactive|The inactivity timer is already counting down.');
+		if (this.resetTimer && user && (this.resetUser === user.userid || (user in this.game.players && this.resetUser === '+') || this.resetUser === '~')) {
```

Comment:
Yeah, that was my guess. Perhaps `timerSetByUser` is clearer.

Why are you testing `user` if you have code above that crashes if `user` isn't truthy?